sdk/typescript/src/cryptography/signature.ts

```diff
@@ -68,7 +68,12 @@ export function toParsedSignaturePubkeyPair(
 		SIGNATURE_FLAG_TO_SCHEME[bytes[0] as keyof typeof SIGNATURE_FLAG_TO_SCHEME];
 
 	if (signatureScheme === 'MultiSig') {
-		return decodeMultiSig(serializedSignature);
+		try {
+			return decodeMultiSig(serializedSignature);
+		} catch (e) {
+			// Legacy format multisig do not render. 
+			return [];
```

Comment:
We should throw here so the error gets rendered

Reply:
do we want to include a custom message here ?